rand_distr/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rand_distr"
-version = "0.2.2"
+version = "0.4.0"
```

Comment:
Why are we skipping 0.3?

Reply:
It was already published. Don't know why the version didn't get bumped here.